lib/output.js

```diff
@@ -978,6 +979,16 @@ function OutputStream(options) {
         }
     });
 
+    PARENS(AST_BigInt, function(output) {
+        var p = output.parent();
+        if (p instanceof AST_PropAccess && p.expression === this) {
+            var value = this.getValue();
+            if (value < 0 || /^0/.test(make_num(value, true))) {
```

Comment:
If we change `value` to be a `BigInt`, then we'll need to change this to: `value < 0n`

Or if we want to avoid using new syntax in Terser, we could also use: `value < BigInt(0)`